std/experimental/logger/filelogger.d

```diff
@@ -21,17 +21,36 @@ class FileLogger : Logger
       file can not be opened for writting an exception will be thrown.
       lv = The $(D LogLevel) for the $(D FileLogger). By default the
       $(D LogLevel) for $(D FileLogger) is $(D LogLevel.all).
+      createFileNameFolder = if true and fn contains a folder name, this
```

Comment:
Definitely use `Flag`. We've essentially decided that all new Phobos APIs that take `bool` parameters to take `Flag`s instead.